airbyte-integrations/bases/base-normalization/.gitignore

```diff
@@ -4,6 +4,7 @@ dbt-project-template/models/generated/
 dbt-project-template/test_output.log
 dbt_modules/
 secrets/
+dist/
```

Comment:
seems you already committed and pushed some binary files:
`airbyte-integrations/bases/base-normalization/dist/normalization-0.0.0-py3.7.egg`

can you delete it from git?

Reply:
You added a `.pyc` file in your PR too